certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+                "convert_to": {
+                  "type": "string",
+                  "description": "The type of file you want to get back",
+                  "title": "What do you want to convert it to?",
+                  "enum": [
+                    "JSON",
+                    "Excel 97-2004 (.xls)",
+                    "CSV",
+                    "XML",
+                    "YAML"
+                  ]
+                }
+              },
+              "required": [
+                "convert_to",
+                "convert",
+                "return_type"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/convert/file/csv/": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "converted": {
+                  "type": "string",
+                  "description": "The converted file from a CSV file",
+                  "title": "Converted File"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your request. Check the error message for more details.",
+                  "title": "Bad Request"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your authentication. Try refreshing the connection or check if your subscription is still valid.",
+                  "title": "Unauthorized"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was an unexpected error. The development team was notified.",
+                  "title": "Unexpected error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Convert a CSV File",
+        "description": "Converts a CSV file to the supported (XLS, XLSX, XML, JSON, YAML) file type",
+        "operationId": "ConvertCSVFile",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "return_type": {
+                  "type": "string",
+                  "description": "return_type",
```

Comment:
@mtgomes descriptions should be valid sentences. More information can be found in [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
@mtgomes Please remove the "_" from the description. all descriptions should be valid sentences. please fix all descriptions in your swagger file.